ConfigSpace/conditions.pyx

```diff
@@ -169,26 +168,21 @@ cdef class EqualsCondition(AbstractCondition):
 
     def __init__(self, child: Hyperparameter, parent: Hyperparameter,
                  value: Union[str, float, int]) -> None:
-        """
-        Hyperparameter ``child`` is conditional on the ``parent`` hyperparameter
+        """Hyperparameter ``child`` is conditional on the ``parent`` hyperparameter
         being *equal* to ``value``.
 
-        Example
-        -------
+        .. code:: python
 
-        >>> import ConfigSpace as CS
-        >>> import ConfigSpace.hyperparameters as CSH
-        >>> cs = CS.ConfigurationSpace()
-        >>> a = CSH.CategoricalHyperparameter('a', choices=[1, 2, 3])
-        >>> b = CSH.UniformFloatHyperparameter('b', lower=1., upper=8., log=False)
-        >>> cs.add_hyperparameters([a, b])
-        [a, Type: Categorical, Choices: {1, 2, 3}, Default: 1, b, Type: ...]
+            cs = ConfigurationSpace({
+                "a": [1, 2, 3],
+                "b": (1.0, 8.0)
+            })
 
-        make *b* an active hyperparameter if *a* has the value 1
+            # make *b* an active hyperparameter if *a* has the value 1
+            cond = EqualsCondition(b, a, 1)
+            cs.add_condition(cond)
 
-        >>> cond = CS.EqualsCondition(b, a, 1)
-        >>> cs.add_condition(cond)
-        b | a == 1
+            # b | a == 1
```

Comment:
The previous formatting would run doctest on the examples. Does the new formatting do the same?

Reply:
No, there's no doctest run. This was basically done to remove the `>>>` in the generated docs and try to make it as clean looking as possible. This does possibly introduce some inaccuracy between the docs and the output a user might receive if we do functional changes.

I'm okay with putting them back in but it does make the docs a little more "crowded" looking for the simple examples.

Reply:
How about we just move those tests to the tests directory?

Reply:
Those things are already tested (I hope)